### ISSUE TITLE
Email notification for absorption rates at nmz

### ISSUE THREAD

**Issue by jamesol1** [runelite/runelite#10723]
I and many others train our stats at nmz using abosproption potions. On a certain tile you can remain AFK for up to 60 minutes aslong as you don't die. Is there a way you could allow the option of runelite sending an email to X address when you for instance go under your absoroption threshold amount? Using remote accessing apps i can train away from my computer quite easily. It just gets tricky managing it though when something pops up and you forget about it.

If there's any other way of getting runelite to message for instance discord maybe that would pop up on my phone that would work too.

**Comment by Nightfirecat** [comment]
I do not think we want to add email functionality to the client. I suspect you can likely get some kind of functionality like this using a product like [IFTTT](https://ifttt.com/) or [PushSafer](https://www.pushsafer.com/).
